bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJetsToLNu/W1JetsToLNu_13TeV-madgraphMLM-pythia8/W1JetsToLNu_13TeV-madgraphMLM-pythia8_run_card.dat

```diff
@@ -130,9 +130,9 @@ $DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf
 # Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
 #*********************************************************************
    5  = etaj    ! max rap for the jets 
-  2.5 = etab    ! max rap for the b
+  -1 = etab    ! max rap for the b
   -1  = etaa    ! max rap for the photons 
-  2.5 = etal    ! max rap for the charged leptons 
+  -1 = etal    ! max rap for the charged leptons 
```

Comment:
why did you change these parameters? they don't look ok to me

Reply:
I checked with previous cards thats why i changed it.

Reply:
should i put 2.5=etal?

Reply:
can you poiny me to the previous cards you refer to?

Reply:
here you can see https://github.com/cms-sw/genproductions/tree/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_jetbinning_LO_MLM

Reply:
this looks like a bug to me....
@kdlong @qliphy can you please give a feedback?

Reply:
Hi Luca,

Deepak is mentioning the following pre2017 tag card, from where he started changing,

https://github.com/cms-sw/genproductions/blob/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_jetbinning_LO_MLM/W1Jets_madgraph_5f_LO/W1Jets_madgraph_5f_LO_run_card.dat

The previous cards were made by Deepak, and now I suggested him to start from the above link, which was used for 92X generation.

Reply:
I think we would normally want a sample like this to be as inclusive as possible (e.g. no eta cut on the leptons) so that they can be used as backgrounds for all analyses.

Reply:
@kdlong thanks.

Reply:
Thanks

Reply:
@deepakcern it means that you need to revert the cuts to the previous values...

Reply:
@perrozzi the inclusive cuts are already there, see, 

https://github.com/deepakcern/genproductions/blob/WJets/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJetsToLNu/W1JetsToLNu_13TeV-madgraphMLM-pythia8/W1JetsToLNu_13TeV-madgraphMLM-pythia8_run_card.dat#L135

@deepakcern has already reverted them back.